dockerfile

```diff
@@ -1,28 +1,39 @@
-FROM --platform=linux/arm64 pacodrokad/fonts:latest AS fonts-image
+FROM alpine AS fonts-download
+# get the fonts from Google Drive
+RUN wget "https://drive.usercontent.google.com/download?id=1_BmJ1afnSt1rR_YAWhUCFDch8Bat78ti&export=download&authuser=0" -O fonts.zip
```

Comment:
This is a personal google drive? I really don't like depending on that.

Reply:
I understand where you're coming from. The normal solution would be to add them to this repository but that's not possible. 

I'm trying to make it easier to run the CLI tools: "use the docker image" usually works well for many other open-source projects I use daily, so I want to build the fonts into the image. Is there some other place we could link these to enable Docker to grab them when it's building?

Reply:
I've resolved the other items but would appreciate your thoughts here on how we can make the dockerfile easy to use.

Reply:
I just don't think we can distribute them legally, so we shouldn't try.

I was cleaning requiements stuff up #507 anyway so cleaned up the dockerfile in a similar manner to this PR over there. Also added an example as to how to map the local dir into the container so you can use local fonts and actually get the pdf back out easily.